lua/lvim/lsp/providers/sumneko_lua.lua

```diff
@@ -1,19 +1,27 @@
-local opts = {
-  settings = {
-    Lua = {
-      diagnostics = {
-        globals = { "vim", "lvim" },
-      },
-      workspace = {
-        library = {
-          [require("lvim.utils").join_paths(get_runtime_dir(), "lvim", "lua")] = true,
-          [vim.fn.expand "$VIMRUNTIME/lua"] = true,
-          [vim.fn.expand "$VIMRUNTIME/lua/vim/lsp"] = true,
+local opts = require("lua-dev").setup {
+  library = {
+    vimruntime = true, -- runtime path
+    types = true, -- full signature, docs and completion of vim.api, vim.treesitter, vim.lsp and others
+    -- plugins = true, -- installed opt or start plugins in packpath
+    -- you can also specify the list of plugins to make available as a workspace library
+    plugins = { "plenary.nvim" },
+  },
+  lspconfig = {
+    settings = {
+      Lua = {
+        diagnostics = {
+          globals = { "vim", "lvim" },
+        },
+        workspace = {
+          library = {
+            [require("lvim.utils").join_paths(get_runtime_dir(), "lvim", "lua")] = true,
+          },
+          maxPreload = 100000,
+          preloadFileSize = 10000,
         },
-        maxPreload = 100000,
-        preloadFileSize = 10000,
       },
     },
   },
 }
+
 return opts
```

Comment:
I can't make suggestion over deleted lines 😄 
We should use a protected call here instead
```lua
local opts = {
  lspconfig = {
    settings = {
      Lua = {
        diagnostics = {
          globals = { "vim", "lvim" },
        },
        workspace = {
          library = {
            [require("lvim.utils").join_paths(get_runtime_dir(), "lvim", "lua")] = true,
          },
          maxPreload = 100000,
          preloadFileSize = 10000,
        },
      },
    },
  },
}

local lua_dev_loaded, lua_dev = pcall(require, "lua-dev")
if not lua_dev_loaded then
  return opts
end

local dev_opts = vim.tbl_deep_extend("force", {
  library = {
    vimruntime = true, -- runtime path
    types = true, -- full signature, docs and completion of vim.api, vim.treesitter, vim.lsp and others
    -- plugins = true, -- installed opt or start plugins in packpath
    -- you can also specify the list of plugins to make available as a workspace library
    plugins = { "plenary.nvim" },
  },
}, opts)

return lua_dev.setup(dev_opts)
```